.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: scala
+scala:
+  - 2.10.6
+  - 2.11.11
+  - 2.12.3
+sudo: false
+script:
```

Comment:
@ayushi3004 You don't need to override the `script`. By default it would always do `sbt test` which is what we want. `clean` is redundant because travis always starts off with clean slate unless we cache something.